arangod/IResearch/IResearchDataStore.cpp

```diff
@@ -1295,25 +1112,18 @@ irs::IndexWriterOptions IResearchDataStore::getWriterOptions(
   }
   // initialize commit callback
   options.meta_payload_provider = [this](uint64_t tick, irs::bstring& out) {
-    const auto version = irs::numeric_utils::hton32(
-        static_cast<std::underlying_type_t<SegmentPayloadVersion>>(
-            SegmentPayloadVersion::TwoStageTick));
-
-    // call from commit under lock _commitMutex (_dataStore._writer->Commit())
-    // update current stage commit tick
-    auto lastCommittedTick =
-        _commitStageOne ? &_lastCommittedTickOne : &_lastCommittedTickTwo;
-    *lastCommittedTick = std::max(*lastCommittedTick, tick);
-    // convert to BE and write low tick
-    tick = irs::numeric_utils::hton64(uint64_t{
-        _commitStageOne ? _lastCommittedTickTwo : _lastCommittedTickOne});
+    // convert version to BE
+    const auto v = irs::numeric_utils::hton32(
+        static_cast<uint32_t>(SegmentPayloadVersion::TwoStageTick));
```

Comment:
Don't we have to change version?

Reply:
No, because now I use second version and write same low and high tick, probably we can increment it, and write only signle tick, but I don't think it's really better